### DISCUSSION TITLE
Umlautprobleme

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#140] [General]
Hallo,

wir haben - sicherlich vernachlässigbare - Umlautprobleme. Hier zum Beispiel

{"ver":"1.0.0","nam":{"fn":"**MüllèČÇ"**,"gn":"Ängölikä"},"dob":"1990-01-01","v":[{"tg":"840539006","vp":"1119305005","dn":1,"sd":2,"dt":"2021-06-23","mp":"EU/1/20/1528","ma":"ORG-100030215","id":"123456789"}]}

wird zu

![Bildschirmfoto 2021-06-23 um 21 05 12](https://user-images.githubusercontent.com/2109844/123154080-b6967b00-d466-11eb-9a92-ee15345ae8f9.png)

Müssen wir spezielle Codepages nutzen?

**Comment by thinkberg** [comment]
Ist das PDF aus dem backend oder mit dem template lokal erzeugt?

**Reply by marcusthierfelder** [reply]
Das ist via backend erzeugt worden.

**Reply by thinkberg** [reply]
Wir haben das Problem identifiziert (font ohne entsprechende Glyphen). Wir werden die Templates jetzt noch einmal ueberarbeiten und einen OSS Font mit grosser Abdeckung uebernehmen. Dann wird das Template auch veroeffentlicht.

**Reply by marcusthierfelder** [reply]
supi, vielen dank
